### PR TITLE
docs: Add documentation for Qdrant integration

### PR DESCRIPTION
**Pull request by jirispilka**
Attached are screenshots for Qdrant. It is very similar to the Pinecone integration. 

⚠️ Note that there is an extra step since Qdrant is not listed in the console's integrations yet, but it will be added soon.

cc @mtrunkat 

Screenshot_20240618_105753
Screenshot_20240618_105821
Screenshot_20240618_105843

### CODE REVIEW

File: sources/platform/integrations/ai/qdrant.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+slug: /integrations/qdrant
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+**Learn how to integrate Apify with Qdrant to transfer crawled into the Qdrant vector database.**
+
+---
+
+[Qdrant](https://qdrant.io) is a high performance managed vector database that allows users to store and query dense vectors for next generation AI applications such as recommendation systems, semantic search, and retrieval augmented generation (RAG).
```
Comment (TC-MO) [inline sources/platform/integrations/ai/qdrant.md:15]: did you mean qdrant.tech? .io throws error in lychee and I cannot find them at this URL

Comment (jirispilka) [reply sources/platform/integrations/ai/qdrant.md]: > did you mean qdrant.tech? .io throws error in lychee and I cannot find them at this URL

Thanks @TC-MO  . My bad, I run replace pinecone -> qdrant and I also replaced URLs 🤦🏻

File: sources/platform/integrations/ai/qdrant.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+title: Qdrant integration
+description: Learn how to integrate Apify with Qdrant to feed data crawled from the web into the Qdrant vector database.
+sidebar_label: Qdrant
+sidebar_position: 4
+slug: /integrations/qdrant
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+**Learn how to integrate Apify with Qdrant to transfer crawled into the Qdrant vector database.**
```
Comment (mtrunkat) [inline sources/platform/integrations/ai/qdrant.md:11]:
```suggestion
**Learn how to integrate Apify with Qdrant to transfer crawled data into the Qdrant vector database.**
```

**Review by mtrunkat — APPROVED** [review]
Looks good

**Comment by jirispilka** [comment]
[Slack discussion](https://apify.slack.com/archives/CQ96RHG2U/p1718624622863889) regarding docs for vector databases integrations

File: sources/platform/integrations/ai/qdrant.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+1. For a comprehensive explanation on how to combine Actors to accomplish more complex tasks, refer to the guide on [Actor-to-Actor](https://blog.apify.com/connecting-scrapers-apify-integration/) integrations.
+
+#### Python
+
+Another way to interact with Qdrant is through the [Apify Python SDK](https://docs.apify.com/sdk/python/).
+
+1. Install the Apify Python SDK by running the following command:
+
+    `pip install apify-client`
```
Comment (TC-MO) [inline sources/platform/integrations/ai/qdrant.md:63]: just one small suggestion

```suggestion
    ```py
    pip install apify-client
    ```
```

**Review by TC-MO — APPROVED** [review]
LGTM, just one small suggestion

**Comment by TC-MO** [comment]
You can disregard lychee errors, those are expected when adding new page due to the way how lychee is setup

File: sources/platform/integrations/ai/qdrant.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+The Apify integration for Qdrant enables you to export results from Apify Actors and Dataset items into a specific Qdrant collection.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following:
+
+- A [Qdrant database](https://qdrant.tech) set up.
+- A Qdrant URL to the database and Qdrant API token.
+- An [OpenAI API key](https://openai.com/index/openai-api/) to compute text embeddings.
```
Comment (Anush008) [inline sources/platform/integrations/ai/qdrant.md:25]:
```suggestion
- An [OpenAI API key](https://openai.com/index/openai-api/) to compute text embeddings. The actor can also be configured to use [Cohere](https://cohere.com/) embedding models.
```

Comment (jirispilka) [reply sources/platform/integrations/ai/qdrant.md:25]: @Anush008 I'm sorry, I missed your comment when merging. I will include it in the next update. 

We are re-working the integration section in console.apify.com and we will need to re-work this doc anyway.

**Pull request merged**